lib/internal/Magento/Framework/App/Bootstrap.php

```diff
@@ -279,7 +279,6 @@ protected function assertMaintenance()
         if (null === $isExpected) {
             return;
         }
-        $this->initObjectManager();
         /** @var \Magento\Framework\App\MaintenanceMode $maintenance */
         $this->maintenance = $this->objectManager->get(\Magento\Framework\App\MaintenanceMode::class);
```

Comment:
I think this could also be removed :) Maintenance is now also initialized in constructor.